app/controllers/blogs_controller.rb

```diff
@@ -0,0 +1,53 @@
+class BlogsController < ApplicationController
+  before_action :load_blog, except: %i(index new create)
+  before_action :logged_in_user, except: %i(new create show)
+
+  def index
+    @blogs = Blog.all
+  end
+
+  def new
+    @blog = current_user.blogs.build
+    @districts_name = []
+    District.all.each do |d|
+      @districts_name.push d.name
+    end
```

Comment:
```suggestion
    @districts_name = District.pluck :name 
```
từ line 11 đến line 14 nhé